optimum/habana/transformers/models/clip/modeling_clip.py

```diff
@@ -1,5 +1,6 @@
 from typing import Optional, Tuple, Union
 
+from optimum.habana.transformers.models.modeling_all_models import Matmul
```

Comment:
```suggestion
from ..modeling_all_models import Matmul
```
Let's use relative imports to stay consistent with other HF libraries